wordpress-mobile/MediaPicker-iOS
# Fixes for iOS 10

This PR fixes the following issues found on iOS 10:
 - multiple selection not working
 - permissions info plist key on demo app weren't set

How to test:
 - Run the demo app on a iOS 10 device or simulator
 - Check if multiple selection work
 - Test on a device to see if access to camera and video are working


Needs Review: @frosty
